vpc-reachability-analyzer/README.md

```diff
@@ -0,0 +1,8 @@
+# This script is designed to test if subnets have access to public AWS API
+
+To run the script open AWS CloudSell in the region where you plan deploy elastio. And run following commands:
+```
+wget https://raw.githubusercontent.com/elastio/contrib/master/vpc-reachability-analyzer/elastio-vpc-reachability-analyzer.sh
```

Comment:
NOTE: are you sure `wget` is present by default in CloudShell?  If so then this is fine.

Reply:
yes, I tested this in my default cloudshell.